README.md

```diff
@@ -1,33 +1,30 @@
-# R4DS [{BOOK_NAME}]({BOOK_LINK}) Book Club
+# R4DS [Modern Dive](https://moderndive.com/) Book Club
 
-A weekly reading group following the chapters of {BOOK_NAME} by {BOOK_AUTHOR(S)}
+A weekly reading group following the chapters of *Modern Dive* by Chester Ismay and Albert Y. Kim
 
-Visit the [{BOOK_CHANNEL} channel](https://r4ds.io/join) on Slack to join the video chat! 
+Visit the [#book_club-modern-dive](https://r4ds.io/join) channel on Slack to join the video chat! 
 
 This repo contains 3 main folders:
 
 - Presentations: each week a presenter will give a talk summarizing the chapter and/or discussing practical applications of the topic at hand. 
 - Data: Example datasets are stored in this folder to help facilitate practical applications! (For more info, please check out [Data/README.md](Data/README.md))
 - QandA: A collection of questions (and answers) on the chapters that come up during meetings (or between!)
 
-
 Other useful guides: [Repo Maintainers Guide](.guides/repo-maintainers.md)
 
 ### Meeting Schedule 
 
-*Cohort 1 - (started 2020-07-31) - Fridays, 8pm E(D)T*
+*Cohort 1 - (started 2020-08-12) - Wednesdays, 7:30 EDT*
 
-- 2020-08-07 Chapter 3 - Data visualisation: Presenter TBD
-- 2020-08-14 Chapter 4 -  Workflow: basics: Presenter TBD
-- 2020-08-21 Chapter 5 - Data visualisation: Presenter TBD
+- 2020-08-07 Chapter ??? - ???: Presenter TBD
```

Comment:
Chapter numbers and titles are "???" because we may wind up skipping some of the initial chapters. This is something we're planning to discuss tomorrow during the first meeting.

Reply:
Sounds good!